### ISSUE TITLE
cyradm warnings with perl 5.28

### ISSUE THREAD

**Issue by jasontibbitts** [cyrusimap/cyrus-imapd#2620]
I happened to run cyradm on a Fedora 29 machine which has perl 5.28.1 and immediately received the following warnings that I don't think I received under perl 5.26:
```
ἐπιθυμία:~❯ cyradm --user cyrus --auth PLAIN foo
Variable "$cyrref" will not stay shared at /usr/lib64/perl5/vendor_perl/Cyrus/IMAP/Shell.pm line 724.
Variable "$lfh" will not stay shared at /usr/lib64/perl5/vendor_perl/Cyrus/IMAP/Shell.pm line 726.
Duplicate specification "server|s=s" for option "s"
```
None of these appear to cause any problems.  I have my perl knowledge swapped out at the moment so I haven't looked into the best ways to fix these.

**Comment by jeffgoh** [comment]
F30 running perl 5.28 has this same problems. I concur with elliefm here - https://github.com/cyrusimap/cyrus-imapd/issues/2747#issuecomment-491652440 and have temporarily removed showacl from cyrus/IMAP/Shell.pm to fix first two errors.  I'm looking for a solution to #3 now and will update once I do.

**Comment by jeffgoh** [comment]
This fixes the "Duplicate specification - ignorecase makes short options 's' and 'S' the same.  This actually is actually an important fix, I think - not sure what happens if you use both -s and -S.
-S is systemrc and -s is server.

RCS file: ./RCS/Shell.pm,v
retrieving revision 1.2
diff -r1.2 ./Shell.pm
63a64
> use Getopt::Long qw(:config no_ignore_case bundling bundling_override);

**Comment by elliefm** [comment]
Thanks @jeffgoh, that's very helpful!  It's now fixed on the 3.2 and master branches, and will be included in the 3.2.2 release.

I left the "bundling" and "bundling_override" options out, to avoid changing the interface.
